modules/common/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-common",
   "author": "Aragon Association",
-  "version": "0.10.0-beta",
```

Comment:
This might be a minor version, not a patch. Also, update the two changelog's please